packages/react-dom/src/__tests__/ReactUpdates-test.js

```diff
@@ -25,6 +26,10 @@ describe('ReactUpdates', () => {
     ReactTestUtils = require('react-dom/test-utils');
     act = require('jest-react').act;
     Scheduler = require('scheduler');
+    ReactFeatureFlags = require('shared/ReactFeatureFlags');
+    ReactFeatureFlags.enableSyncDefaultUpdates = true;
+    ReactFeatureFlags.enableUnifiedSyncLane = true;
+    ReactFeatureFlags.enableUseRefAccessWarning = false;
```

Comment:
do we need these flags for all tests, or they only for specific case(maybe we should move these flags in that test?)

Reply:
no, these need to be cleaned up.